### PR TITLE
Clarify account merging

### PR DESCRIPTION
**Pull request by kbatuigas**
# Description
Clarify what happens when a new email/password account is connected to a VCS account.

# Reasons
[Jira ticket](https://circleci.atlassian.net/browse/DOCTEAM-740)

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/gitlab-integration.adoc [hunk 1]
```diff
@@ -54,7 +54,7 @@ This new organization allows you to manage users and GitLab projects within the
 
 Once you successfully create a new organization, you will be taken to the organization dashboard. Follow the steps in <<#step-two-create-a-new-project,Step Two>> below to set up a new GitLab project.
 
-NOTE: Existing CircleCI users can also opt to sign up via email and password. If you choose to connect a VCS (GitHub or Bitbucket) account that you already have with your new email/password account, your accounts will be merged based on the VCS account name that you used. More information can be found in <<#user-account-integrations,Account integrations>> under the User settings section in this document.
+NOTE: Existing CircleCI users can also opt to sign up with a new email and password account. When you connect the same VCS account to your email/password account, your CircleCI accounts will be merged. More information can be found in <<#user-account-integrations,Account integrations>> under the User settings section in this document.
```
Comment (kbatuigas) [inline jekyll/_cci2/gitlab-integration.adoc:57]: @jessiejjacobson @Frozenfire92 I changed the wording a bit. Does this still convey the correct meaning?

Comment (jessiejjacobson) [reply jekyll/_cci2/gitlab-integration.adoc:57]: Yes. I think that is clean and makes the point. @Frozenfire92 agree?

Comment (Frozenfire92) [reply jekyll/_cci2/gitlab-integration.adoc:57]: Yep, thanks for checking @kbatuigas @jessiejjacobson

Comment (devinmarieb) [reply jekyll/_cci2/gitlab-integration.adoc:57]: I guess adding on here.... the phrase "with a new email" makes it seem like CircleCI is going to ask you to create a CircleCI email just for this purpose OR that you are creating a brand new email just for this purpose? Are we trying to say someone can use an existing email with a new password just for CircleCI?

**Review by devinmarieb — APPROVED** [review]
@kbatuigas Just one comment because reading that I'm a little confused myself.

File: jekyll/_cci2/gitlab-integration.adoc (hunk 1 above)
Comment (kbatuigas) [reply jekyll/_cci2/gitlab-integration.adoc:57]: @devinmarieb ah, I was thinking more along the lines of, there is the "Sign up with Email" option on the signup page, which I believe _first_ creates a brand new email+password account, and then if you connect the VCS later then that's when the new account gets merged into the existing one. I guess we can also just say "sign up using email and password"

**Pull request merged**